spec/content/manifest.spec.yml

```diff
@@ -22,6 +22,20 @@ spec:
               name:
                 description: Name of the field.
                 type: string
+              value:
+                description: >
+                  An optional expected value for the "name" field.
+                oneOf:
+                - type: string
+                  minLength: 1
```

Comment:
Here, I'm forcing that if it is a string, it should not be an empty string.

For instance, this would not be allowed:
```yaml
discovery
  fields:
    - name: event.dataset
      value: ""
```

Would that make sense ?
Depending on that, the validation in EPR could be different.

Reply:
> should we allow to compare a field with an empty string?

I guess this is in theory possible, but I don't know what would be the use case. So feel free to keep this `minLength`.